### DISCUSSION TITLE
Error: Cannot find module '../build/Release/sharp.node

### DISCUSSION THREAD

**Discussion by iababio** [gatsbyjs/gatsby#31661] [Help]
onPreBootstrap/usr/local/lib/node_modules/gatsby-cli/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

## Description

➜  gatsby-starter-personal-blog gatsby develop                              
success delete html and css files from previous builds — 0.006 s
success open and validate gatsby-config — 0.005 s
success copy gatsby files — 0.011 s
⠂ onPreBootstrap/usr/local/lib/node_modules/gatsby-cli/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

Error: Cannot find module '../build/Release/sharp.node'
Require stack:
- /Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-plugin-sharp/node_modules/sharp/lib/constructor.js
- /Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-plugin-sharp/node_modules/sharp/lib/index.js
- /Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-plugin-sharp/index.js
- /Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-transformer-sharp/customize-schema.js
- /Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-transformer-sharp/gatsby-node.js
- /Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby/dist/utils/api-runner-node.js
- /Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby/dist/utils/webpack-modify-validate.js
- /Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby/dist/utils/webpack.config.js
- /Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby/dist/commands/develop.js
- /usr/local/lib/node_modules/gatsby-cli/lib/create-cli.js
- /usr/local/lib/node_modules/gatsby-cli/lib/index.js
- /usr/local/lib/node_modules/gatsby-cli/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:997:19)
    at require (/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-plugin-sharp/node_modules/sharp/lib/constructor.js:10:15)
    at Module._compile (/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/v8-compile-cache/v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:997:19)
    at require (/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-plugin-sharp/node_modules/sharp/lib/index.js:3:15)
    at Module._compile (/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/v8-compile-cache/v8-compile-cache.js:178:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-plugin-sharp/node_modules/sharp/lib/constructor.js',
    '/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-plugin-sharp/node_modules/sharp/lib/index.js',
    '/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-plugin-sharp/index.js',
    '/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-transformer-sharp/customize-schema.js',
    '/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby-transformer-sharp/gatsby-node.js',
    '/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby/dist/utils/api-runner-node.js',
    '/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby/dist/utils/webpack-modify-validate.js',
    '/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby/dist/utils/webpack.config.js',
    '/Users/mac/Lab/Learn/gatsby-starter-personal-blog/node_modules/gatsby/dist/commands/develop.js',
    '/usr/local/lib/node_modules/gatsby-cli/lib/create-cli.js',
    '/usr/local/lib/node_modules/gatsby-cli/lib/index.js',
    '/usr/local/lib/node_modules/gatsby-cli/cli.js'
  ]
}

### Steps to reproduce

Clear steps describing how to reproduce the issue. **Please** link to a reproduction, this makes your issue _much_ easier to diagnose (seriously).

How to Make a Minimal Reproduction: https://www.gatsbyjs.com/contributing/how-to-make-a-reproducible-test-case/

What happened.
=================
gatsby develop

### Environment
macOS

`gatsby info --clipboard`
========================
System:
    OS: macOS 11.4
    CPU: (4) x64 Intel(R) Core(TM) i7-7567U CPU @ 3.50GHz
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 15.7.0 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 7.14.0 - /usr/local/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 91.0.4472.77
    Safari: 14.1.1
  npmPackages:
    gatsby: ^1.9.274 => 1.9.279
    gatsby-image: ^1.0.55 => 1.0.55
    gatsby-link: ^1.6.46 => 1.6.46
    gatsby-plugin-algolia: ^0.0.4 => 0.0.4
    gatsby-plugin-catch-links: ^1.0.24 => 1.0.26
    gatsby-plugin-feed: ^1.3.25 => 1.3.25
    gatsby-plugin-google-analytics: ^1.0.31 => 1.0.31
    gatsby-plugin-jss: ^1.5.14 => 1.5.14
    gatsby-plugin-manifest: ^1.0.27 => 1.0.27
    gatsby-plugin-netlify: ^1.0.21 => 1.0.21
    gatsby-plugin-offline: ^1.0.18 => 1.0.18
    gatsby-plugin-react-helmet: ^2.0.11 => 2.0.11
    gatsby-plugin-react-next: ^1.0.11 => 1.0.11
    gatsby-plugin-react-svg: ^1.1.1 => 1.1.1
    gatsby-plugin-sharp: ^1.6.48 => 1.6.48
    gatsby-plugin-sitemap: ^1.2.25 => 1.2.25
    gatsby-remark-component: ^1.1.3 => 1.1.3
    gatsby-remark-copy-linked-files: ^1.5.37 => 1.5.37
    gatsby-remark-external-links: ^0.0.4 => 0.0.4
    gatsby-remark-images: ^1.5.67 => 1.5.67
    gatsby-remark-prismjs: ^2.0.5 => 2.0.5
    gatsby-remark-responsive-iframe: ^1.4.20 => 1.4.20
    gatsby-remark-smartypants: ^1.4.10 => 1.4.12
    gatsby-source-filesystem: ^1.5.39 => 1.5.39
    gatsby-transformer-remark: ^1.7.44 => 1.7.44
    gatsby-transformer-sharp: ^3.6.0 => 3.6.0
  npmGlobalPackages:
    gatsby-cli: 3.6.0

**Comment by LekoArts** [comment]
You're using a very outdated starter, try running `npm init gatsby` instead.
Also make sure that you have the necessary build tools for `sharp` installed: https://github.com/nodejs/node-gyp#on-macos
